cllc-public-app/Controllers/ContactController.cs

```diff
@@ -479,5 +484,46 @@ private string GetDocumentLocationReferenceByRelativeURL(string relativeUrl)
 
             return result;
         }
+
+        
+        [HttpGet("phs-link/{contactId}")]
+        [AllowAnonymous]
+        public JsonResult Subscribe(string contactId)

```

Comment:
Are you sure this service should be anonymous?

Reply:
This will not be anonymous in the end. I just need it for testing now as the calling component is not having this functionality yet.